gatsby/createPages.js

```diff
@@ -41,16 +41,27 @@ module.exports.createMarkdownPages = async ({ graphql, actions }) => {
 
 					// Convert URLs accessed at "/reactivesearch/vue/SearchBox" -> "/reactivesearch/vue/searchbox"
 					if (
+						node &&
+						node.fields &&
 						node.fields.slug &&
-						node.fields.slug.includes('/docs/reactivesearch/vue/') &&
-						!node.fields.slug.includes('/docs/reactivesearch/vue/v1')
+						node.fields.slug.toLowerCase().startsWith('/docs/reactivesearch/v4')
 					) {
-						// Create a duplicate page for "/reactivesearch/vue/searchbox"
-						const optionsForDuplicatePage = {
+						// Create a page with /v4 and one with /react
+						createPage({
+							...options,
+							path: node.fields.slug.toLowerCase().replace('/v4', '/react'),
+						});
+						createPage(options);
+					} else if (
+						node.fields.slug &&
+						node.fields.slug.toLowerCase().startsWith('/docs/reactivesearch')
```

Comment:
This may create duplicate pages for which the url is already lowercase. eg. `/docs/reactivesearch/react/searchbox`.

Reply:
@SavvyShah Thanks for pointing out, the next commit addresses this